source/MaterialXRenderGlsl/GlslRenderer.cpp

```diff
@@ -321,6 +321,20 @@ void GlslRenderer::save(const FilePath& filePath)
     }
 }
 
+ImagePtr GlslRenderer::saveImage()
```

Comment:
Hmm. Lost a comment. This should be reusable by save(). Currently you have 2 sets of code almost the same. The code in save() is more complete. Then have save() call saveImage().